package.json

```diff
@@ -23,9 +23,11 @@
     "bower": "^1.7.7",
     "cheerio": "^0.19.0",
     "cookie-parser": "^1.4.1",
+    "denodeify": "^1.2.1",
```

Comment:
I've been [using](https://github.com/Financial-Times/google-amp/blob/532b541f4ef26dd96ec3376abd28e30f2ff0d78a/server/lib/compile-scss.js#L3) [`@quarterto/promisify`](https://gist.github.com/quarterto/726a8947687cb33ca907). Don't mind which we go for as long as it's consistent